mobile/function/get_sponsors.js

```diff
@@ -28,6 +28,8 @@ export const GetSponsors = () => {
             .catch((error) => console.error(error))
             .finally(() => setLoading(false));
     }, [fetchData]);
+    
+    console.log("Loading sponsor data");
```

Comment:
ook console.log weg halen tenzij je die gebruikt die er met een reden is natuurlijk